### DISCUSSION TITLE
post/bilibili-share-vip/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#26] [Announcements]
# post/bilibili-share-vip/

上次买的一年半 B 站大会员到期了, 感觉一共没追几部番真是血亏, 而且新番一鸽再鸽质量还飘忽不定(三体说的就是你), 完全没有继续充值大会员的欲望. 之前网上冲浪的时候无意中看到过一个所谓的 “B 站大会员解析”, 实际上是通过服务器端共享的 Cookie 去访问获取视频链接的接口, 解析出完整的视频链接替换掉网页的试看视频. 于是直接找同学分享给我了一个大会员账号的 Cookie, 尝试复刻出

https://blog.lyc8503.net/post/bilibili-share-vip/

**Comment by Yizhixixi** [comment]
链接似乎已经失效（检查一下？）

**Reply by lyc8503** [reply]
不对这篇文章提供任何技术支持，仅是自用脚本思路分享~

（B站更新后方法失效也是正常的）

**Comment by s060908** [comment]
脚本未失效，仅为无法自动切换画质，需要手动切换画质
